test/check-engine.js

```diff
@@ -18,17 +18,17 @@ test("node version too old", function (t) {
 })
 
 test("npm version too old", function (t) {
-  var target = { engines: { npm: "1.3.6" }}
-    c(target, "1.4.2", "0.2.1", false, true, function (err) {
+  var target = { engines: { npm: "1.4.6" }}
```

Comment:
This should actually be a range. Doing an exact match here doesn't actually test if anything is too old, just that it's different.